app/templates/master_events/table_manager.html

```
@@ -229,14 +229,16 @@ <h2 class="mb-0">Tabulkový manažer</h2>
                     title="Odhlásit">✕</button>
             {% endif %}
           {% elif row.can_manage %}
-            <select class="form-select form-select-sm tm-assign-select"
+            <select class="form-select form-select-sm tm-assign-select user-conflict-picker"
                     data-spot-id="{{ spot.id }}"
                     id="assign_spot_{{ spot.id }}">
               <option value="">— přiřadit —</option>
               {% for user in row.eligible_users if user.id not in row.assigned_user_ids %}
-              <option value="{{ user.id }}">{{ user.name }}</option>
+              {% set _has_conflict = user.id in row.conflicted_user_ids %}
+              <option value="{{ user.id }}"{% if _has_conflict %} data-conflict="1" data-conflict-details='{{ row.user_conflict_details.get(user.id|string, []) | tojson }}'{% endif %}>{% if _has_conflict %}⚠️ {% endif %}{{ user.name }}</option>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)